apps/ios/fastlane/Fastfile

```diff
@@ -53,4 +55,77 @@ platform :ios do
       release_notes: release["body"]
     )
   end
+
+  lane :ci_match_install do
+    setup_circle_ci
+
+    get_app_store_connect_api_key
+
+    match(
+      type: "development",
+      app_identifier: app_identifier,
+      readonly: true,
+      verbose: true,
+      force_for_new_devices: false
+    )
+
+    match(
+      type: "enterprise",
+      app_identifier: app_identifier,
+      readonly: true,
+      verbose: true,
+      force_for_new_devices: false
+    )
+  end
+
+  desc "Install existing match certs and profiles without updating/overwriting"
+  lane :match_install do
+    get_app_store_connect_api_key
+
+    match(
+      type: "development",
+      app_identifier: app_identifier,
+      readonly: true,
+      verbose: true,
+      force_for_new_devices: false
+    )
+
+    match(
+      type: "enterprise",
+      app_identifier: app_identifier,
+      readonly: true,
+      verbose: true,
+      force_for_new_devices: false
+    )
+  end
+
+  desc "Update and overwrite match certs and profiles if needed - destructive and may require other devs to match_install"
+  lane :match_update do
+    match(
+      type: "development",
+      app_identifier: app_identifier,
+      readonly: false,
+      verbose: true,
+      force_for_new_devices: true
+    )
+
+    match(
+      type: "enterprise",
+      app_identifier: app_identifier,
+      readonly: false,
+      verbose: true,
+      force_for_new_devices: true
+    )
+  end
+
+  desc "Get local api key - assumes you have downloaded the api key to the project directory"
```

Comment:
This description is out of date now that we are only using environment variables.

Reply:
Oh makes sense. Seems like less is more here with comments so just deleted it.